features/pecl/info-extensions.feature

```diff
@@ -12,7 +12,7 @@ Feature: download and install PECL extensions
       """
     And the output should contain:
       """
-      Downloading: 100%
+      Downloading (100%)
       """
```

Comment:
I removed all of them as it prevents such thing to happen again (they can change that again...). It is ok as this does not prevent the test itself to actually test a behavior and its success or failure.